### PR TITLE
docs: 1607 update compile guide

### PR DESCRIPTION
**Pull request by aceforeverd**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

add extra notice for those that want compile over latest commit

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: docs/zh/deploy/compile.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 此节介绍在官方编译镜像 [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql) 中编译 OpenMLDB。镜像内置了编译所需要的工具和依赖，因此不需要额外的步骤单独配置它们。关于基于非 docker 的编译使用方式，请参照下面的 [编译详细说明](#编译详细说明) 章节。
 
+:bulb: 关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 `hybridsql:0.4.4` 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 `main` 分支的代码，则需要拉取 `hybridsql:latest` 版本镜像。
```
Comment (lumianph) [inline docs/zh/deploy/compile.md]: this emoji 💡  not working in MyST parser, you can remove it.

File: docs/en/compile.md [hunk 2]
```diff
@@ -8,23 +8,25 @@ Build OpenMLDB
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
+:bulb: Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
```
Comment (lumianph) [inline docs/en/compile.md]: also this ":bulb:" (not working in the MyST parser), you may remove it

**Review by lumianph — COMMENTED** [review]
you may remove this emoji ":bulb:"

**Comment by aceforeverd** [comment]
> you may remove this emoji "💡"

fixed

**Review by lumianph — APPROVED** [review]

File: docs/en/compile.md [hunk 3]
```diff
@@ -8,23 +8,25 @@ Build OpenMLDB
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
+Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
```
Comment (lumianph) [inline docs/en/compile.md]: This setion demonstrate
=>
This section demonstrates

Comment (aceforeverd) [reply docs/en/compile.md]: fixed

File: docs/en/deploy/compile.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
-Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
+Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrates compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
```
Comment (lumianph) [inline docs/en/deploy/compile.md]: setion => section

**Pull request merged**